### PR TITLE
Update event-029.md

### PR DESCRIPTION
**Pull request by catsmith**
Adding time

### CODE REVIEW

**Comment by Chilipp** [comment]
thanks @catsmith! @eileen-kuehn, please let me know when you created the registration event on indico

File: _events/talks/event-029.md [hunk 1]
```diff
@@ -10,11 +10,15 @@ authors:
...
       index: 1
+time:
+    - - start: 2021-01-26T14:00:00Z
+        end: 2021-01-26T14:30:00Z
 author: *speaker
 category: talks
 language: English
 prerequisites: "None. Interest in numerical simulations will make the talk more relevant to the audience."
 date: 2020-11-03
+last_modified_at: 2020-11-30
```
Comment (eileen-kuehn) [inline _events/talks/event-029.md:21]:
```suggestion
last_modified_at: 2020-11-30
registration_url: https://indico.scc.kit.edu/event/2260/
```

**Comment by eileen-kuehn** [comment]
@Chilipp, I just added the *registration_url*, should be ready to go 👍

**Review by eileen-kuehn — APPROVED** [review]
Ready to go from my side 👍

**Review by Chilipp — APPROVED** [review]
thank you both =)

**Pull request merged**